### PR TITLE
Use es client

### PR DESCRIPTION
**Pull request by leggsimon**
@i-like-robots

### CODE REVIEW

File: server/checks/in-sync.js [hunk 1]
```diff
@@ -1,26 +1,23 @@
-const signedFetch = require('signed-aws-es-fetch');
-const sync = require('../lib/sync');
+const client = require('../lib/client.js');
+const sync = require('../lib/sync.js')
```
Comment (i-like-robots) [inline server/checks/in-sync.js]: Can you tweak the autocomplete to skip the file extension?

File: test/server/lib/sync-spec.js [hunk 2]
```diff
@@ -1,41 +1,53 @@
...
 	afterEach(() => {
+		sandbox.reset();
 		nock.cleanAll();
 	});
 
 	it('doesn\'t check CAPI if there are no differences in ids', () => {
-		euESRequest = euESRequest.reply(200, fixture1)
-		usESRequest = usESRequest.reply(200, fixture1)
+		client.eu.search.resolves(fixture1)
+		client.us.search.resolves(fixture1)
```
Comment (i-like-robots) [inline test/server/lib/sync-spec.js]: Is this `sinon-as-promised`? Not seen it before

**Comment by i-like-robots** [comment]
I think it looks a bit cleaner - what do you think?

File: test/server/lib/sync-spec.js (hunk 2 above)
Comment (leggsimon) [inline test/server/lib/sync-spec.js]: Yep. Syntactic sugar for:
```js
client.eu.search.returns(Promise.resolve(fixture1))
```

File: server/checks/in-sync.js (hunk 1 above)
Comment (leggsimon) [inline server/checks/in-sync.js]: Ah dammit. I removed it in `sync.js`. I'll remove this and see if I can tweak settings

**Comment by leggsimon** [comment]
Yeah looks much nicer. Can't wait to kill `nock` though!

**Comment by leggsimon** [comment]
Is this g2g then? Maybe wait until tomorrow?

File: server/checks/in-sync.js [hunk 3]
```diff
@@ -1,26 +1,23 @@
...
+		query: {
+			bool: {
+				must_not: [{ regexp: { provenance: '.*acast.*' } }]
+			}
+		},
+		size: 0
+	}
+
+	return client[region].count({
+		index: 'v3_api_v2',
```
Comment (i-like-robots) [inline server/checks/in-sync.js]: One little thing missing - `searchType: 'count'` (see the query string on the old URL)

**Comment by i-like-robots** [comment]
LGTM - just one small comment then 🚢

File: server/checks/in-sync.js (hunk 3 above)
Comment (leggsimon) [inline server/checks/in-sync.js]: Remember we changed that to `client.count()`

**Pull request merged**